### PR TITLE
Add additional Plesk add cPanel information in cloud servers FAQ

### LINKED ISSUE #1272: On boarding team doc suggestion - How to install and use CPanel
#Relates to #1264 
SME: Tyler Lutz
I know this one may be a community item but just in case: CPANEL and Rackspace – how to install and utilize CPANEL high level overview

### PR DESCRIPTION
**Pull request by kylelaffoon**
- Fixes #1272

### CODE REVIEW

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-238b36b1c3/how-to/). :bow:

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-500a82652f/how-to/)

**Comment by kylelaffoon** [comment]
Emailed Dusty Simnoni to ask if only the managed service level accounts are affected. He replied: "The highlighted section is accurate, but we might want to make it a bit stronger to emphasize the point.  The usage of Plesk or  cPanel is not supported and strongly discouraged for customers under the Managed Operations SLA as it interferes with our server administration.
 
Customers of lower SLAs frequently do utilize cPanel and Plesk with no problems." 

I modified the verbiage with his stronger statement in this commit.

File: content/cloud-servers/cloud-servers-faq.md [hunk 1]
```diff
@@ -292,7 +292,8 @@ Yes, the Rackspace [Cloud Control Panel](https://mycloud.rackspace.com/)
 is a web-based management interface for restarting your cloud server,
 starting support discussions, viewing stats, and scheduling snapshots.
 However, we do not offer a control panel like Plesk or cPanel. You're
-free to install such packages for your own cloud server.
+free to install such packages for your own cloud server. If you are installing cPanel, be sure to install on a clean (empty) server. See [CentOS/RHEL - Installing cPanel & WHM 11.24](https://community.rackspace.com/products/f/25/t/6798) for additional information.
+The usage of Plesk or  cPanel is not supported and strongly discouraged for customers under the Managed Operations SLA as it interferes with our server administration. See [Using Plesk or cPanel with the Managed Operations Service Level Agreement](https://community.rackspace.com/products/f/25/t/4888) for additional information.
```
Comment (cyrichardson) [inline content/cloud-servers/cloud-servers-faq.md]: extra space between or and cPanel

**Comment by cyrichardson** [comment]
@kylelaffoon A minor nit in comment

**Comment by kylelaffoon** [comment]
Thanks, @cyrichardson . Fixed the nit :)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-5f543a1b82/how-to/)

**Comment by kylelaffoon** [comment]
Too many problems. Abandonning this and starting over.

**Pull request merged**